x/concentrated-liquidity/position.go

```diff
@@ -391,6 +391,10 @@ func (k Keeper) deletePosition(ctx sdk.Context,
 	return nil
 }
 
+func (k Keeper) CreatePosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, tokensProvided sdk.Coins, amount0Min, amount1Min sdk.Int, lowerTick, upperTick int64) (positionId uint64, actualAmount0 sdk.Int, actualAmount1 sdk.Int, liquidityDelta sdk.Dec, lowerTickResult int64, upperTickResult int64, err error) {
```

Comment:
@stackman27 @p0mvn For ProtoRev to be able to create positions, I had to create a new exported `CreatePosition` method, previously a `CreatePosition` testing helper method was created in export_test.go but protorev doesn't have access to that method. 

If there's a better way to get access to this ability I'm open to changing this to what ever is desired from y'alls end.

Reply:
If we are doing this I think it just makes sense to export the original createPosition. This will end up being confusing later on down the road imo

Reply:
@czarcas7ic Sounds good, I'll create a separate PR that just exports it and changes where it's being called from, then can rebase this PR on top of that once merged

Reply:
PR up: https://github.com/osmosis-labs/osmosis/pull/5893, will ping in slack once all CI finishes and r4r